lib/manager/components/GeneralSettings.js

```diff
@@ -130,7 +132,7 @@ export default class GeneralSettings extends Component<Props, State> {
                     value={this._getFormValue('name')}
                     name={'name'}
                     disabled={disabled}
-                    onChange={this._onChange} />
+                    onChange={(e) => this._onChange(e, 'name')} />
```

Comment:
This change can be reverted once we switch to using `target.name`

Reply:
You betcha.